Makefile

```diff
@@ -116,6 +117,7 @@ update_dev_tools:
 get_tools:
 	@echo "--> Installing tools"
 	$(MAKE) -C scripts get_tools
+	go get -u golang.org/x/tools/cmd/goimports
```

Comment:
@ValarDragon @greg-szabo can we get this binary via the Makefile in `/scripts`?

Reply:
we can do that if we change the format of command getting tools